### PR TITLE
Pytest start optimization 

### PR DESCRIPTION
**Pull request by Arthur264**
Speed up tests duration

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for aedf38955290914c4a267f46e248e73c87598a86 (Arthur264/saleor @ patch-1)
Base comparison is 6bb11e9c573b251df13a02576a190f53a1985fd4.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         10	         10	              0
  complete checkout                          	          8	          8	              0
  create checkout                            	          5	          5	              1

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         18	         18	              4
  retrieve product attributes                	          9	          9	              0

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         51	         51	              3
  retrieve variant list                      	         23	         23	              9

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

**Comment by maarcingebala** [comment]
Thanks for opening the PR @Arthur264. What is the time difference when running tests with and without this change?

**Comment by maarcingebala** [comment]
It seems Black is still failing. @fowczarek We may want to add this change when we start working on our tests improvements.

File: setup.cfg [hunk 1]
```diff
@@ -15,7 +15,7 @@ exclude_lines =
     if TYPE_CHECKING:
 
 [tool:pytest]
-addopts = -n auto --vcr-record-mode=none --ds=tests.settings
+addopts = -n auto --vcr-record-mode=none --ds=tests.settings --nomigrations
```
Comment (NyanKiyoshi) [inline setup.cfg]: You could directly use `--reuse-db` instead, no?

**Comment by Arthur264** [comment]
Hi,
During pytest start, all migrations running and it takes around of 30s-1m
**--nomigrations** and **--reuse-db** accelerate this process (**--nomigrations** automatically generate only one migration from the current state of the model)
Resume (on my local machine):
Before changes (all tests):
~155s
After changes:
~90s

**Pull request closed without merging**